spaceship/models/mission.py

```diff
@@ -22,3 +22,24 @@ class Mission(BaseModel):
   @hybrid_property
   def is_active(self):
     return self.deleted_at == None
+
+  @hybrid_property
+  def start_time(self):
+    return self.started_at.format('dddd [the] Do [of] MMMM')
+  
+  @hybrid_property
+  def is_over(self):
+    now = pendulum.now('UTC')
+    mission_end_time = self.started_at.add(weeks=self.duration_in_weeks)
+    return now >= mission_end_time
+
+  @hybrid_property
+  def is_running(self):
+    now = pendulum.now('UTC')
+    mission_end_time = self.started_at.add(weeks=self.duration_in_weeks)
+    return now >= self.started_at and now <= mission_end_time
```

Comment:
does this make missions 8 days long instead of 7?

Reply:
should probably be < rather than <=

Reply:
< makes sense to me--8 days does not, should be adding exactly 7*24hours. Tested with the "?now=YYYY-MM-DD parameter in the url.